docs/application/vscode-ext/web.md

```diff
@@ -79,142 +51,151 @@ To download emulator images, you can use Tizen Package Manager or Tizen Emulator
-4. To run the application on the emulator, enter **Run Tizen App** in the **Command Palette**.
+3. Create and launch an emulator instance in the Emulator Manager.
+4. To deploy your application to the target, type **Install Tizen App** in the **Command Palette**.
+5. To run the application on the emulator, type **Run Tizen App** in the **Command Palette**.
 
-To install your project:
+To install project:
 
-  1. Open **Command Palette** and enter **Install Tizen App**.
-  2. Enter the folder name of the project you want to install. (Folder name, not path)
-  3. When the installation is successfully completed, the installation app is generated. Also, you will see a build success log in the **terminal** at the bottom of the vscode.
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Install Tizen App** and press **Enter**.
+3. Enter the folder name of the project you want to install.
+    > **Note**
+    >
+    > Ensure that you enter the name of the folder and not the path.
 
-  ![Install your tizen app](media/tizen_web14.png)
+    ![Install your tizen app](media/tizen_web14.png)
 
 > **Note** 
 > 
-> If you have a Tizen device and you want to deploy the application to it, you must first generate, and then add a certificate.
+> If you have a Tizen device and you want to deploy the application to it, then you must first generate a certificate and then add the certificate.
 
-To run your project:
+To run project:
 
   1. Open **Command Palette** and enter **Run Tizen App**.
-  2. Enter the folder name of the project you want to run. (Folder name, not path)
-  3. When the run is complete, it succeeds, and there is successful generation of launch app popup. Also, you will see a build success log in the terminal at the bottom of the vscode.
-
+  2. Enter the folder name of the project that you want to run. 
+      > **Note**
+      >
+      > Ensure that you enter the name of the folder and not the path.
+  3. When the installation is successfully completed, the installation app is generated. Also, you will see a build success log in the **terminal** at the bottom of the vscode.
   ![Run Your Tizen App](media/tizen_web15.png)
 
-> **Note**  
->
-> If you have a Tizen device and you want to deploy the application to it, you must first generate, and then add a certificate.
->
-> In addition, vscode terminal displays successful launch log messages:
-> 
-> 1. debug 0 means normal launch.
-> 2. debug 1 means launching an Inspector available.
+        > **Note**  
+        >
+        > If you have a Tizen device and you want to deploy the application to it, then you must first generate a certificate and then add the certificate.
+        >
+        > In addition, `vscode` terminal displays successful launch log messages:
+
+- debug `0` means normal launch.
+- debug `1` means launching an inspector available from command line.
```

Comment:
Please right align line 197 and 198.
It should be part of the note.

Reply:
done.